libpod/kube.go

```diff
@@ -288,6 +289,16 @@ func newServicePortState() servicePortState {
 	}
 }
 
+func TruncateKubeAnnotation(str string) string {
+	str = strings.TrimSpace(str)
+	if utf8.RuneCountInString(str) < define.MaxKubeAnnotation {
```

Comment:
Okay, one more. The only [spec](https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/) I can find (I didn't look too hard) mentions 63 characters but doesn't actually indicate if we're talking new-style Unicode or old-style ASCII C chars. Does anyone know how to figure this one out?

Reply:
IDK, but I"m interested in seeing the answer....

Reply:
It also inconsistent since cmd/podman/play/kube.go the byte len is checked and not utf8 len.